elkm1_lib/notify.py

```diff
@@ -38,4 +38,4 @@ def notify(self, notify_type: str, notify_parameters: dict[str, Any]) -> None:
             try:
                 observer(**notify_parameters)
             except Exception as exc:  # pylint: disable=broad-except
-                LOG.exception(exc)
+                LOG.exception(f"For observer {observer} params {notify_parameters}: {exc}",exc_info=True)
```

Comment:
Break this out into a separate PR. That will allow for focused testing. Perhaps add a test (I notice I missed added a test for this).

Reply:
Ok, will do.  Yes, that exception drove me crazy for a long time since I assumed it was something I broke in my library, but was really caused by parameter calling changing for the kc, as soon as I added more info to the exception it was obvious where my issue was.